scripts/simulate_and_process_l1_files.sh

```diff
@@ -15,7 +15,7 @@ python -m make_romanisim_input_catalog
 # 2 - create roman_photoz simulated catalog (using COSMOS templates and lephare.prepare method)
 # No additional photometric errors added to the simulated catalog; these will come in during
 # the imaging simulations.
-python -m roman_photoz.create_simulated_catalog --output-path .
+python -m roman_photoz.create_simulated_catalog --output-path . --refresh-lib-mag
```

Comment:
Is --refresh-lib-mag required or only useful when one is using new versions of lephare / new configuration options?

Reply:
Sounds good, thanks.